scripts/quota_log.py

```diff
@@ -35,12 +47,19 @@ def disk_quota():
 
 def cpu_hours():
     current_time = datetime.datetime.now()
-
-    # script that runs on uppmax
-    uq = subprocess.Popen(["/sw/uppmax/bin/projinfo", '-q'], stdout=subprocess.PIPE)
+    try:
+        # script that runs on uppmax
+        uq = subprocess.Popen(["/sw/uppmax/bin/projinfo", '-q'], stdout=subprocess.PIPE)
+    except Exception, e:
+        logging.error(e.message)
+        exit()
```

Comment:
Or even just remove the try / except block? As the logging is going to the terminal anyway, so no difference to the python raise really.